package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "inverted-index",
+  "version": "1.0.0",
+  "description": "Inverted index application using elastic search",
+  "main": "index.js",
+  "dependencies": {
+    "browser-sync": "^2.18.8",
+    "browserify": "^2.3.0",
+    "express": "^4.15.2",
+    "file-api": "^0.10.4",
+    "gulp": "^3.9.1",
+    "gulp-concat": "^2.6.1",
+    "gulp-connect": "^5.0.0",
+    "gulp-open": "^2.0.0",
+    "gulp-uglify": "^2.1.2",
+    "jasmine-expect": "^3.7.0",
+    "object.values": "^1.0.4",
+    "path": "^0.12.7"
+  },
+  "devDependencies": {
+    "coveralls": "^2.12.0",
+    "eslint": "^3.17.1",
+    "eslint-config-airbnb": "^14.1.0",
+    "eslint-config-airbnb-base": "^11.1.1",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsdoc": "^2.4.0",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
+    "gulp-browserify": "^0.5.1",
+    "gulp-concat": "^2.6.1",
```

Comment:
Since 'gulp-concat' is in the dependencies, there is no need to add it to devDependencies again